scijava/scijava-discovery-therapi/src/test/java/module-info.test

```diff
@@ -0,0 +1,2 @@
+--add-reads
+	org.scijava.discovery.therapi=org.scijava
```

Comment:
What is this file? `module-info.test`? I've not heard of this, and cannot find anything with a quick web search. There is no explanation in the corresponding commit about what it is. Is it just cruft that should be deleted?